css/styles.css

```diff
@@ -386,3 +386,128 @@ li {
   margin-right: 2rem;
   color: #acb7c3;
 }
+
+/** FORM SECTION **/
+.form-section {
+  background-color: #6070ff;
+  border-top-left-radius: 10rem;
+  padding-top: 5rem;
+  position: relative;
+}
+
+.form-rectangle {
+  position: absolute;
+  width: 5.4rem;
+  height: 3.6rem;
+  top: 2rem;
+  right: 0;
+}
+
+.form-half-circle {
+  position: absolute;
+  width: 5rem;
+  height: 8.9rem;
+  top: 4rem;
+  right: 0;
+}
+
+.form-img1 {
+  position: absolute;
+  width: 2.1rem;
+  height: 2.1rem;
+  top: 5rem;
+  left: 15rem;
+}
+
+.form-heading {
+  text-align: center;
+  font-size: 4rem;
+  font-weight: 700;
+  color: #fff;
+  margin-top: 6rem;
+}
+
+.form-paragraph {
+  font-size: 2rem;
+  font-weight: 300;
+  text-align: center;
+  padding-top: 3rem;
+  line-height: 2.8rem;
+  color: #fff;
+}
+
+form {
+  margin-top: 2rem;
+}
+
+fieldset {
+  border: none;
+  margin-top: 2rem;
+  display: flex;
+  flex-direction: column;
+}
+
+.text,
+.email,
+.text-area {
+  padding: 2rem 1rem;
+  font-size: 1.7rem;
+  color: #172b4d;
+  font-weight: 400;
+  display: block;
+  width: 100%;
+  border-radius: 1rem;
+  margin-bottom: 1.5rem;
+  border: none;
+}
+
+.submit {
+  background-color: #fff;
+  color: #6070ff;
+  font-size: 1.7rem;
+  font-weight: 500;
+  padding: 2rem 1rem;
+  border-radius: 1rem;
+  border: none;
+  text-transform: capitalize;
+  transition: 0.5s ease;
+  margin-bottom: 4rem;
+  align-self: flex-start;
+}
+
+.submit:hover {
+  cursor: pointer;
+  background-color: #6070ff;
+  color: #fff;
+  border: 0.2rem solid #fff;
+}
+
+.submit:active {
+  padding: 2.5rem 1.5rem;
+  -webkit-box-shadow: 0 0 1.3rem 1rem #fff;
+  box-shadow: 0 0 1.3rem 1rem #fff;
+}
```

Comment:
- [OPTIONAL ] The Button interactions such as hover, active (when it's pressed), disabled and enable (default state) are styled as shown in the Figma template. Please use the some CSS properties from the Figma design look ⬇️ 

![buttongood](https://user-images.githubusercontent.com/79058364/165333032-7d562e97-53c3-4614-a452-1f680d6fcf41.png)